List returns error when not logged in

```
You must be loggen in
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x164ae08]

goroutine 1 [running]:
github.com/bf2fc6cc711aee1a0c2a/cli/cmd/kafka.runList(0xc0002ff340, 0x1d3c1c8, 0x0, 0x0)
        /Users/wtrocki/Projects/mas/mas-devexp/rhmas/cmd/kafka/list.go:45 +0x1
```

--- a/cmd/kafka/massdk.go
+++ b/cmd/kafka/massdk.go
@@ -2,6 +2,7 @@ package kafka
 
 import (
 	"fmt"
+	"os"
 	"strings"
 
 	mas "github.com/bf2fc6cc711aee1a0c2a/cli/client/mas"
@@ -18,12 +19,12 @@ func BuildMasClient() *mas.APIClient {
 
 	if err != nil {
 		fmt.Println("Error loading configuration")
-		return nil
+		os.Exit(1)
 	}
 
 	if cfg.AccessToken == "" && cfg.RefreshToken == "" {
-		fmt.Println("You must be loggen in")
-		return nil
+		fmt.Println("You must be loggen in. To do so use the `rhmas login` command")
+		os.Exit(1)
 	}
 
 	urlSegments := strings.Split(cfg.URL, "://")

